README.md

```diff
@@ -31,3 +31,12 @@ git clone --depth=1 https://github.com/rust-lang/rust.vim.git ~/.vim/bundle/rust
 
 1. Add `NeoBundle 'rust-lang/rust.vim'` to `~/.vimrc`
 2. Re-open vim or execute `:source ~/.vimrc`
+
+## Enabling autoformat
+
+This plugin can optionally format your code using [rustfmt][rfmt] every time a
+buffer is written. Simple put `let g:rust_fmt_autosave = 1` in your `.vimrc`.
```

Comment:
Hi.  I'm trying this out, and it works great!

Just a quick note: here you have `rust_fmt_autosave` as the bool to check for autosave, but lookup uses `g:rustfmt_autosave` (note the extra underscore).